examples/components/mysql/service.jsonnet

```diff
@@ -1,25 +0,0 @@
-local kap = import "lib/kapitan.libjsonnet";
-local inventory = kap.inventory();
-
-local namespace = inventory.parameters.namespace;
-local instance_name = inventory.parameters.mysql.instance_name;
-
-
-{
-  apiVersion: "v1",
-  kind: "Service",
-  spec: {
-    ports: [
-      { name: "mysql", port: 3306, target_port: "mysql" },
```

Comment:
Seems like this file was deleted, but it was just moved to `examples/kubernetes` and this line was changed to:
`target_port` -> `targetPort`